### ISSUE TITLE
`attempt to index field 'factory-circuit-connector' (a nil value)` error when starting server on 1.5.14

### ISSUE THREAD

**Issue by lediur** [GregorSamsanite/nullius#8]
Thanks for your work on the mod!

I just installed the most recent update (1.5.14, released yesterday) and my server is reporting the following error on startup:

```
4.582 Loading mod nullius 1.5.14 (data-updates.lua)
   4.753 Error Util.cpp:83: Failed to load mod "nullius": __nullius__/prototypes/override_mod.lua:208: attempt to index field 'factory-circuit-connector' (a nil value)
stack traceback:
        __nullius__/prototypes/override_mod.lua:208: in main chunk
        [C]: in function 'require'
        __nullius__/data-updates.lua:5: in main chunk
```

The server then fails to complete startup, so I need to revert to 1.5.13 to start the server successfully.

Mods I have installed:

```
AutoTrash_5.3.13.zip
BottleneckLite_1.2.4.zip
DisplayPlates_1.4.1.zip
Factorissimo2_2.5.3.zip
FactorySearch_1.10.4.zip
FluidMustFlow_1.3.1.zip
GDIW_1.1.0.zip
Honk_5.0.0.zip
'Induction Charging_2.0.2.zip'
'Inventory Sensor_1.10.6.zip'
Milestones_1.3.9.zip
ModuleInserter_5.2.4.zip
MultipleUnitTrainControl_0.4.15.zip
PipeVisualizer_1.0.4.zip
QuickItemSearch_2.1.5.zip
RateCalculator_2.4.4.zip
RealisticDecorationCleanup_1.1.1.zip
RecipeBook_3.4.5.zip
Renamer_2.1.7.zip
Robot256Lib_1.1.4.zip
Tapeline_2.0.8.zip
TaskList_0.4.2.zip
TrainGroups_1.1.7.zip
Warehousing_0.5.7.zip
aai-signal-transmission_0.4.7.zip
alien-biomes_0.6.7.zip
angelspetrochem_0.9.22.zip
angelsrefining_0.12.2.zip
angelssmelting_0.6.19.zip
assemblyanalyst_1.1.11.zip
bobinserters_1.1.5.zip
boblibrary_1.1.5.zip
boblogistics_1.1.5.zip
botReplacer_1.1.0.zip
discovery_tree_1.0.4.zip
even-distribution_1.0.10.zip
factoryplanner_1.1.59.zip
flib_0.11.2.zip
jetpack_0.3.5.zip
miniloader_1.15.6.zip
// this is a server-specific mod that changes some train behavior, shouldn't interact with nullius
niflheim-vehicle-tweaks_1.0.0.zip
nullius_1.5.14.zip
production-analyst_1.1.8.zip
rso-mod_6.2.20.zip
safefill_1.0.10.zip
stdlib_1.4.7.zip
textplates_0.6.10.zip
train-pubsub_1.1.13.zip
underground-pipe-pack_1.1.2.zip
whats-missing_0.1.14.zip
```

**Comment by domis86** [comment]
can you start new game with 1.5.13, make save and send it here? Will be easier to test.

**Comment by lediur** [comment]
Sure thing. Here's the save:

[nullius-1.5.13-github-issue-8.zip](https://github.com/GregorSamsanite/nullius/files/9849575/nullius-1.5.13-github-issue-8.zip)

Here's also the server-specific mod, since I haven't uploaded it to the mod portal:

[niflheim-vehicle-tweaks_1.0.0.zip](https://github.com/GregorSamsanite/nullius/files/9849576/niflheim-vehicle-tweaks_1.0.0.zip)

**Comment by lediur** [comment]
Checking back in to confirm that I updated to 1.5.15 and the server starts successfully! Thanks for the quick fix.

**Comment by lediur** [comment]
Hmm, upon loading into the game, something else might be wrong since all the Nullius entities (buildings, etc) seem to have disappeared. I'll post in Discord about this.

**Comment by GregorSamsanite** [comment]
Sounds like the latest issue is from loading a post-conflict save and is resolved, so I'm marking this as fixed by the 1.5.15 update.
